src/key_info_managers/mod.rs

```diff
@@ -221,13 +233,13 @@ impl KeyInfoManagerClient {
     pub fn remove_key_info(
         &self,
         key_triple: &KeyTriple,
-    ) -> parsec_interface::requests::Result<KeyInfo> {
+    ) -> parsec_interface::requests::Result<(Vec<u8>, Attributes)> {
```

Comment:
Since nothing broke in the code changing this (everything compiled successfully), I think all callers of this method do:
```rust
let _ = client.remove_key_info(key_triple)?;
```
ignoring the returned value.

For that reason I think it would be fine to make this method return nothing, ie `Result<()>`?

If we wanted to return the key info, it would be best to replace the `Vec<u8>` with the generic deserialised type (ie `T` in the method `get_key_id`). Return nothing seems like a reasonable simplification for now, since this is not used.